LevelComponents/Level.cpp

```diff
@@ -9,77 +9,133 @@
+                                    "\xe3\x81\xaf\xe3\x81\xb2\xe3\x81\xb5\xe3\x81\xb8\xe3\x81\xbb"
+                                    "\xe3\x81\xbe\xe3\x81\xbf\xe3\x82\x80\xe3\x82\x81\xe3\x82\x82"
+                                    "\xe3\x82\x84\xe3\x82\x86\xe3\x82\x88\xe3\x82\x89\xe3\x82\x8a"
+                                    "\xe3\x82\x8b\xe3\x82\x8c\xe3\x82\x8d\xe3\x82\x8f\xe3\x82\x92"
+                                    "\xe3\x82\x93\xe3\x81\x81\xe3\x81\x83\xe3\x81\x85\xe3\x81\x87"
+                                    "\xe3\x81\x89\xe3\x82\x83\xe3\x82\x85\xe3\x82\x87\xe3\x81\xa3"
+                                    "\xe3\x81\x8c\xe3\x81\x8e\xe3\x81\x90\xe3\x81\x92\xe3\x81\x94"
+                                    "\xe3\x81\x96\xe3\x81\x98\xe3\x81\x9a\xe3\x81\x9c\xe3\x81\x9e"
+                                    "\xe3\x81\xa0\xe3\x81\xa2\xe3\x81\xa5\xe3\x81\xa7\xe3\x81\xa9"
+                                    "\xe3\x81\xb0\xe3\x81\xb3\xe3\x81\xb6\xe3\x81\xb9\xe3\x81\xbc"
+                                    "\xe3\x81\xb1\xe3\x81\xb4\xe3\x81\xb7\xe3\x81\xba\xe3\x81\xbd"
+                                    "\xe3\x82\xa2\xe3\x82\xa4\xe3\x82\xa6\xe3\x82\xa8\xe3\x82\xaa"
+                                    "\xe3\x82\xab\xe3\x82\xad\xe3\x82\xaf\xe3\x82\xb1\xe3\x82\xb3"
+                                    "\xe3\x82\xb5\xe3\x82\xb7\xe3\x82\xb9\xe3\x82\xbb\xe3\x82\xbd"
+                                    "\xe3\x82\xbf\xe3\x83\x81\xe3\x83\x84\xe3\x83\x86\xe3\x83\x88"
+                                    "\xe3\x83\x8a\xe3\x83\x8b\xe3\x83\x8c\xe3\x83\x8d\xe3\x83\x8e"
+                                    "\xe3\x83\x8f\xe3\x83\x92\xe3\x83\x95\xe3\x83\x98\xe3\x83\x9b"
+                                    "\xe3\x83\x9e\xe3\x83\x9f\xe3\x83\xa0\xe3\x83\xa1\xe3\x83\xa2"
+                                    "\xe3\x83\xa4\xe3\x83\xa6\xe3\x83\xa8\xe3\x83\xa9\xe3\x83\xaa"
+                                    "\xe3\x83\xab\xe3\x83\xac\xe3\x83\xad\xe3\x83\xaf\xe3\x83\xb2"
+                                    "\xe3\x83\xb3\xe3\x82\xa1\xe3\x82\xa3\xe3\x82\xa5\xe3\x82\xa7"
+                                    "\xe3\x82\xa9\xe3\x83\xa3\xe3\x83\xa5\xe3\x83\xa7\xe3\x83\x83"
+                                    "\xe3\x82\xac\xe3\x82\xae\xe3\x82\xb0\xe3\x82\xb2\xe3\x82\xb4"
+                                    "\xe3\x82\xb6\xe3\x82\xb8\xe3\x82\xba\xe3\x82\xbc\xe3\x82\xbe"
+                                    "\xe3\x83\x80\xe3\x83\x82\xe3\x83\x85\xe3\x83\x87\xe3\x83\x89"
+                                    "\xe3\x83\x90\xe3\x83\x93\xe3\x83\x96\xe3\x83\x99\xe3\x83\x9c"
+                                    "\xe3\x83\x91\xe3\x83\x94\xe3\x83\x97\xe3\x83\x9a\xe3\x83\x9d"
+                                    "\xe3\x83\xb4"; // Hiragana and Katakana
+                                    //"\x27" //'
+        unsigned char tmpstr2[10] = "\xe3\x80\x81\xe3\x80\x82\xe2\x80\x94"; // 、。—
```

Comment:
Is there a reason that these different sections are split into multiple encoded strings?

Reply:
they have to be in the correct order.